presto-hive/src/test/java/io/prestosql/plugin/hive/s3/MockAmazonS3.java

```diff
@@ -18,20 +18,30 @@
 import com.amazonaws.services.s3.model.CannedAccessControlList;
 import com.amazonaws.services.s3.model.GetObjectMetadataRequest;
 import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.ListObjectsRequest;
+import com.amazonaws.services.s3.model.ObjectListing;
 import com.amazonaws.services.s3.model.ObjectMetadata;
 import com.amazonaws.services.s3.model.PutObjectRequest;
 import com.amazonaws.services.s3.model.PutObjectResult;
 import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import com.amazonaws.services.s3.model.StorageClass;
+
+import java.util.Date;
 
 import static java.net.HttpURLConnection.HTTP_OK;
 
 public class MockAmazonS3
         extends AbstractAmazonS3
 {
+    private static final String STANDARD_OBJECT_KEY = "test/standard";
```

Comment:
Let's inline these since they are only used in one place (and the usage is self-descriptive)